net/radsecproxy/src/opnsense/mvc/app/controllers/OPNsense/RadSecProxy/forms/general.xml

```diff
@@ -0,0 +1,51 @@
+<form>
+
+    <field>
+        <id>radsecproxy.general.Enabled</id>

```

Comment:
Sometimes Enabled starts up- and lowcase, just cosmetic.

Reply:
Changed to lowercase.